lua/impatient/cachepack.lua

```diff
@@ -1,16 +1,26 @@
 local ffi = require('ffi')
 
 -- using double for packing/unpacking numbers has no conversion overhead
-local sizeof_c_double = ffi.sizeof("double")
-local c_double = ffi.typeof("double[1]")
+if jit.arch == "arm" then
+  ffi.cdef [[
+    typedef int buffer_t;
```

Comment:
Is buffer the right term here? I would have thought something like `number_t` would be a more fitting name since it is just a type we need to store lua numbers.